Docs/ProjectSpec.md

```diff
@@ -747,3 +769,28 @@ schemes:
       customArchiveName: MyTarget
       revealArchiveInOrganizer: false
 ```
+
+## Swift Package
+Swift packages are defined at a project level, and then linked to individual targets via a [Dependency](#dependency).
+
+> Note that Swift Packages don't work in projects with configurations other than Debug and Release
```

Comment:
Can we add a link to https://bugs.swift.org/browse/SR-10927?

Reply:
Done